.maestro/security_tests/3_-_AddressBarSpoof,_appschemes.yaml

```diff
@@ -0,0 +1,30 @@
+appId: com.duckduckgo.mobile.android.debug
```

Comment:
Looking at https://github.com/duckduckgo/Android/actions/runs/7126639716, it only ran the first two tests.
You need to set the `appId` to `com.duckduckgo.mobile.android` for tests 3-7.